core/unit_test/CMakeLists.txt

```diff
@@ -205,6 +205,7 @@ foreach(Tag Threads;Serial;OpenMP;Cuda;HPX;OpenMPTarget;OpenACC;HIP;SYCL)
       ViewOfClass
       ViewResize
       View_64bit
+      MDSpan
```

Comment:
I am sure whoever made the list in alphabetical order will appreciate your inserting here

Reply:
I will make sure they appreciate me